spec/controllers/v1.2/stageaction_controller_spec.rb

```diff
@@ -1,6 +1,72 @@
-RSpec.describe Api::V1x2::StageactionController do
+RSpec.describe Api::V1x2::StageactionController, :type => [:v1x2, :request, :controller] do
+  let(:key) { create(:random_access_key, :access_key => 'unique-123', :approver_name => 'Joe Smith') }
+  let!(:approval_request) { create(:request, :with_tenant, :with_context, :random_access_keys => [key]) }
+
+  describe 'GET /stageaction/:id' do
+    it 'returns 200' do
+      get "#{api_version}/stageaction/#{key.access_key}"
+
+      expect(controller.instance_variable_get(:@request)).to eq(approval_request)
+      expect(controller.instance_variable_get(:@approver)).to eq('Joe Smith')
```

Comment:
These expectations don't really match what the `it` block claims this is doing. Looking at the actual code, it appears the `show` is empty. Should we actually be expecting a 200 here or is this just kind of a no-op?

Reply:
Don't know why 500 is always returned here, but in my env it returns 200.